boards.txt

```diff
@@ -1767,6 +1767,222 @@ esp32wroverkit.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+aventen_s3_sync.name=Aventen S3 Sync
+aventen_s3_sync.vid.0=0x303a
+aventen_s3_sync.pid.0=0x1001
+## Based upon ESP32-S3 Dev Board
+
+aventen_s3_sync.bootloader.tool=esptool_py
+aventen_s3_sync.bootloader.tool.default=esptool_py
+
+aventen_s3_sync.upload.tool=esptool_py
+aventen_s3_sync.upload.tool.default=esptool_py
+aventen_s3_sync.upload.tool.network=esp_ota
+
+aventen_s3_sync.upload.maximum_size=1310720
+aventen_s3_sync.upload.maximum_data_size=327680
+aventen_s3_sync.upload.flags=
+aventen_s3_sync.upload.extra_flags=
+aventen_s3_sync.upload.use_1200bps_touch=false
+aventen_s3_sync.upload.wait_for_upload_port=false
+
+aventen_s3_sync.serial.disableDTR=false
+aventen_s3_sync.serial.disableRTS=false
+
+aventen_s3_sync.build.tarch=xtensa
+aventen_s3_sync.build.bootloader_addr=0x0
+aventen_s3_sync.build.target=esp32s3
+aventen_s3_sync.build.mcu=esp32s3
+aventen_s3_sync.build.core=esp32
+aventen_s3_sync.build.variant=aventen_s3_sync
```

Comment:
CI is failing, as you have mixed naming in variant.
variant in board.txt file is `aventen_s3_sync` but variant folder is `Aventen_S3_Sync`.

Please align the naming.
```suggestion
aventen_s3_sync.build.variant=Aventen_S3_Sync
```